containers/daap-python-base/src/var/task/data_platform_paths.py

```diff
@@ -19,6 +19,7 @@
 from datetime import datetime
 from typing import NamedTuple
 from uuid import UUID, uuid4
+from typing import Optional
```

Comment:
```suggestion
from typing import NamedTuple, Optional
from uuid import UUID, uuid4
```
linting

Reply:
Thanks

Reply:
FYI You can hit 'Commit suggestion' to add the change